README.md

```diff
@@ -46,6 +49,20 @@
 9. have a way for users to contact the folks operating the site (email, Discord, etc.).
 10. notify users beforehand when your TOS or privacy policy is changing.
 11. not be a copy of other's bot lists (open sourced lists, including lists that are edited or modified from other's lists).
+12. have a better way of handling errors, instead of showing console errors, show a custom 500 status code page.
+13. load fast enough, and use not a lot of time and/or internet data to load your site for the first time.
+14. have a caching system in place.
+15. not have a custom keybind to stop browser dev tools from loading.
+16. save users from getting xss attacks.
+17. sanitize bot's long description.
+18. allow custom css.
+19. store markdown, render it as html on user's request.
```

Comment:
```suggestion
```
Neither of these feel like items a list _should_ do, up to them what features they decide to offer, a list can be perfectly functional and useful without them

Reply:
Maybe have 19 more be like `have a simple format for long description (markdown, HTML, ...)`?